### PR TITLE
feat(Makefile): add install target

### PR DESCRIPTION
**Pull request by vdice**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -71,5 +73,9 @@ publish:
 	fi
 	az storage blob upload-batch -d porter/mixins/$(MIXIN)/$(PERMALINK) -s $(BINDIR)/$(VERSION)
 
+install:
```
Comment (carolynvs) [inline Makefile]: This assumes that $PORTER_HOME exists and has a mixin dir, etc. We plan on having a `porter init` command that makes this directory and sets it up. So I think it's good to go for now, once we have `porter init` we can update `make install` to call `porter init` if the directory is missing.

Comment (carolynvs) [reply Makefile]: oh it also assumes that the helm mixin has already been installed, and the helm directory under PORTER_HOME/mixins/ exists which I think is a bit off?

How about one change: this does a `mkdir -p` for that helm directory? Let me know what you think.

Comment (vdice) [reply Makefile]: Good call; added (also switched to use of the [install](https://linux.die.net/man/1/install) utility)

Comment (vdice) [reply Makefile]: In addition to `cp`-ing the binary, [install](https://unix.stackexchange.com/questions/94679/what-is-the-purpose-of-the-install-command) also sets executable permissions and can optionally invoke [strip](https://en.wikipedia.org/wiki/Strip_(Unix)) to slim it down.

Comment (carolynvs) [reply Makefile]: Nice! 👍

**Pull request merged**